extras/parse_travis_log.py

```diff
@@ -191,13 +191,41 @@ def print_static_analysis(d):
                 print(INDENT + ' | ' + 2 * INDENT +
                       ' - ' + k3 + ' : ' + str(v3))
 
+def count_warnings_errors(f):
+    """
+    Counts compiler warnings and errors. Stops when reading '+make install'.
```

Comment:
These changes are from a commit (81a7b8048e5744) in an earlier pull request., which was pulled into my pull request by a merge from master. I can't comment on this code. Feel free to open an issue to explain the problem.